src/utils/internal.ts

```diff
@@ -241,7 +247,8 @@ export function createClaim(
   claimType: string,
   jurisdiction: Falsyable<string>,
   middlewareScope: Falsyable<MiddlewareScope>,
-  cddId: Falsyable<string>
+  cddId: Falsyable<string>,
+  customClaimTypeId: Falsyable<BigNumber>
```

Comment:
Seems easy to forget to update the last arg of an object when it can be `..., null, null)`. To me the test case using this makes it seem easy to forget to pass in a `customClaimTypeId`. I think a check that throws could help with this API.

Could be nice to have this function accept an object so undefined things don't need to be passed, but I think its OK as is, just worried we'll make a custom claim with `id: undefined` by mistake someday.